packages/flutter/lib/src/physics/friction_simulation.dart

```diff
@@ -90,14 +92,20 @@ class FrictionSimulation extends Simulation {
 
   @override
   bool isDone(double time) => dx(time).abs() < tolerance.velocity;
+
+  @override
+  String toString() => '${objectRuntimeType(this, 'FrictionSimulation')}(cₓ: ${_drag.toStringAsFixed(1)}, x₀: ${_x.toStringAsFixed(1)}, dx₀: ${_v.toStringAsFixed(1)})';
```

Comment:
Will these print nicely in terminals? Especially worries whether Windows will print these as garbage...

Reply:
should be no worse than what we're doing today with the Animation toStrings, right?